core/baseline-core-v1.0-psd01.md

```diff
@@ -819,65 +825,100 @@ The identifiers and identity utilized in a BPI MUST be controlled by its Princip
 
+[[CR6]>[O1]](#cr6>o1) Testability: DID methods such as ["did:ethr"](https://github.com/decentralized-identity/ethr-did-resolver/blob/master/doc/did-method-spec.md) only support the method where only the principal owner can assign other DIDs for purposes of authentication and assertion to third parties. And in special circumstances control of identifiers, such as control over someone's device that has been assigned the DID. 
+
 #### **[R35]**	
 Private keys MUST be stored and/or transmitted securely.
 
+[[R35]](#r35) Testability: The private keys of identifiers must never be stored on a BPI. The set of private keys not used to control identifiers in the BPI should be stored outside of the BPI with the BPI user. An example ef secure transmission of private keys in the context of a transaction is for a BPI to craft a transaction, send this transaction to a BPI user for signing/authorization, and send this transaction back to the BPI to complete processing. An alternative implementation option is a custodial solution using an established key vault that stores private keys which are separate from the private keys that control identifiers. The private keys contained within the vault are encrypted at rest and only decrypted with a secret known only to the BPI user, for which that private key can be used for signing and encryption/decryption. The key vault should be sealed after every operation or after a pre-set time that should not be longer than a few minutes. 
+
 *No mechanism should exist that would allow a single entity to deprive a Principal Owner of their identifier without their consent. This implies that:*
 
 #### **[CR7]<[D11]**	
 Once a namespace is created within the context of a DPKI, it MUST NOT be possible to destroy it.
 
+[[CR7]>[D11]](#cr7>d11) Testability: The DID method used must not allow for DID deletion to be enacted by anyone.
+
 #### **[CR8]<[D11]**	
 Namespaces in a DPKI utilized by a BPI MUST NOT contain blacklisting mechanisms that would allow anyone to invalidate identifiers that do not belong to them.
 
+[[CR8]>[D11]](#cr8>d11) Testability: The DID method used must not allow blacklisting mechanisms such as suppressing the delivery of DID documents through the universal resolver for specific DIDs.
+
 #### **[CR9]<[D11]**	
 The rules for registering and renewing identifiers in a DPKI utilized by a BPI MUST be transparent and expressed in simple terms.
 
+[[CR9]>[D11]](#cr9>d11) Testability: The DID method used in a BPI is registered in the DID registry and conformant to the [W3C DID Standard](https://www.w3.org/TR/did-core/).
+
 #### **[R36]**	
 If registration is used as security to an expiration policy, the Principal Owner MUST be explicitly and timely warned that failure to renew the registration on time could result in the Principal Owner losing control of the identifier.
 
+[[R36]](#r36) Testability: The DID method used within a BPI does not have an expiration policy.
```

Comment:
this doesn't test the requirement except in the edge case of an non-expiring DID (which is probably itself not a good idea in most cases)